diff -up cryptobone-1.1.1/src/renamesymbols.than cryptobone-1.1.1/src/renamesymbols
--- cryptobone-1.1.1/src/renamesymbols.than	2017-07-13 11:48:58.765722815 +0200
+++ cryptobone-1.1.1/src/renamesymbols	2017-07-13 11:53:46.400922665 +0200
@@ -8,6 +8,9 @@
 # this script must be run in the cryptlib directory before building the
 # shared library
 
+# rename to MASK2h to MASK2h, fixed build issue on powerpc
+sed -i 's/MASK2h1/MASK2h/g' bn/bn_lcl.h
+
 for F in $(find . -type f)
 do
     sed -i 's/BN_/cl_BN_/g' $F
